tests/smoke/test_docbuilder_base.py

```diff
@@ -3,12 +3,10 @@
 from lib.docbuilder import DocumentBuilder
 from lib.data_files_helper import DataFilesHelper
 
-SMOKES_PATH = os.path.abspath('scripts/smoke/templates')
-
 
 @pytest.mark.parametrize(
     'template_path',
-    DataFilesHelper.get_test_files(SMOKES_PATH),
+    DataFilesHelper.get_test_files('scripts/smoke/templates'),
```

Comment:
The paths in the tests are not a good solution.

Select the configuration file to import into the required tests

In case other test sources are connected